rest-api/tools/import_organizations.py

```diff
@@ -323,14 +352,29 @@ def _cleanup_old_entities(self, session, row_list, dry_run):
     if sites_to_remove:
       site_id_list = []
       for site in sites_to_remove:
-        log_prefix = '(dry run) ' if dry_run else ''
         logging.info(log_prefix + 'Deleting old Site no longer in Google sheet: %s', site)
         old_site = self.site_dao.get_by_google_group(site)
+      if old_site and old_site.isObsolete != ObsoleteStatus.OBSOLETE:
         site_id_list.append(old_site.siteId)
         self.deletion_count += 1
+      elif old_site.isObsolete == ObsoleteStatus.OBSOLETE:
```

Comment:
Same as above, checked for old_site is not None.

Reply:
done